File: httpsnippet/src/test/resources/output/node/native/multipart-data.js
var http = require("http");

var options = {"path":"/har","headers":{"content-type":"multipart/form-data"},"hostname":"mockbin.com","method":"POST","port":80};

var req = http.request(options, function (res) {
  var chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.end();

File: httpsnippet/src/test/resources/output/go/native/multipart-data.go
package main

import (
	"fmt"
	"strings"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "http://mockbin.com/har"

	payload := strings.NewReader("foo=Hello World")

	req, _ := http.NewRequest("POST", url, payload)

	req.Header.Add("content-type", "multipart/form-data")

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}
